components/BlurhashImage.vue

```diff
@@ -56,14 +72,40 @@ export default Vue.extend({
       image: ''
     };
   },
+  computed: {
+    getImageHash(): string {
```

Comment:
This is missing a case for thumbs.

Imo, there are three possibilities:

* If it's "backdrop", go: backdrop -> nothing
* If it's "primary", go: primary -> nothing
* If it's "thumb", go: thumb -> backdrop -> nothing

Reply:
I've updated this. I'm think it's correct, but am not 100% sure